test/jdk/jdk/jfr/event/metadata/TestLookForUntestedEvents.java

```diff
@@ -89,7 +89,7 @@ public class TestLookForUntestedEvents {
 
     // Experimental events
     private static final Set<String> experimentalEvents = Set.of(
-        "Flush", "SyncOnValueBasedClass");
+        "Flush", "SyncOnValueBasedClass", "CPUTimeSample", "CPUTimeSampleLoss");
```

Comment:
I think the reason we added an exclusion list for experimental events is so developers wouldn't be forced to write a test. Since you have a test for CPUTimeSample event, I think you can removed it.

(If TestLookForUntestedEvents is not able to find the event test, write the event name in the test)

Reply:
But this leads to fails, because of the loop that initialized the `jfrEventTypes`:
```java
       for (EventType type : FlightRecorder.getFlightRecorder().getEventTypes()) {
            if (type.getAnnotation(Experimental.class) == null) {
                jfrEventTypes.add(type.getName().replace("jdk.", ""));
            }
        }
```
Only events that not experimental are included. But `CPUTimeSample` is experimental.

Reply:
I update the test to check for CPUTimeSample events